react/components/LogoutCustomerSession.js

```diff
@@ -62,20 +69,51 @@ export default class LogoutCustomerSession extends Component {
           </div>
           <div className="bg-white w-100 pa4">
             <div className="vtex-telemarketing__logout-form gray">
-              <div className="flex justify-center pa3 bw1 bb b--silver">
-                <CustomerIcon color={'#828282'} />
-                <div className="pa3">{clientEmail}</div>
-              </div>
-              <div className="flex justify-center">
-                <span className="mt3">
-                  <Button
-                    size="small"
-                    onClick={() => onSetSesssion('')}
-                    isLoading={loading}
-                  >
-                    {translate('telemarketing-logout.button', intl)}
+              <table className="w-100 pa3 bw1 bb b--silver">
```

Comment:
Should not be using tables.

https://www.lifewire.com/dont-use-tables-for-layout-3468941

Reply:
agree